include/internal/libspdm_common_lib.h

```diff
@@ -596,7 +596,7 @@ bool libspdm_calculate_m1m2_hash(void *context, bool is_mut,
  * @retval RETURN_SUCCESS  l1l2 is calculated.
  */
 bool libspdm_calculate_l1l2(void *context, void *session_info,
-                            size_t *l1l2_buffer_size, void *l1l2_buffer);
+                            libspdm_large_managed_buffer_t *l1l2);
```

Comment:
It looks like the scope of this function is entirely in `library/spdm_common_lib/libspdm_com_crypto_service.c` and the header declaration can be removed and the function can be made `static`. @jyao1 Do you think other files will need to access this?